### DISCUSSION TITLE
Way to use inpaint anything or something similar  (segmentation - > inpainting) ?

### DISCUSSION THREAD

**Discussion by a3nima** [lllyasviel/stable-diffusion-webui-forge#1611] [Q&A]
I've been used to work with inpaint anything its fast and works pretty well, if you want to change backgrounds, or stuff and you dont have to draw the mask by hand, which saves a lot of times, also its way more precise. Since Gradio 4 it doesnt work inside forge. So I've been wondering if there is an alternative or other way to get the inpainting mask by segmentation of the image ?

**Comment by YashMohey** [comment]
+1, this was my most used extension and makes inpainting so much easier. Would love to know the alternatives for it.
